### ISSUE TITLE
Could you draw rButtonAura above the Blizzard cooldown swirl/overlay?

### ISSUE THREAD

**Issue by arcadepro** [zorker/rothui#51]
When using rButtonAura for something like Battle Cry it's very hard to see the coloured border because the Blizzard 'darkened swirl' overlay is drawn above it. Image below.

![wowscrnshot_021718_015745](https://user-images.githubusercontent.com/1977760/36337055-a0bf3f78-1386-11e8-99cf-bb34c60af9d5.jpg)

To be honest it's an issue for all spells since the GCD swirl also draws above rButtonAura and leads to weird visuals.

![wowscrnshot_021718_020603a](https://user-images.githubusercontent.com/1977760/36337106-4023202a-1387-11e8-8261-00aefbf0d4e3.jpg)

**Comment by zorker** [comment]
WIll recheck but imo I fixed that using rButtonTemplate. It resets the cooldown frame level to button level. https://github.com/zorker/rothui/blob/master/wow7.0/rButtonTemplate/core.lua#L151

Otherwise the border would not have been visible in my screenshot: http://cdn-wow.mmoui.com/preview/pvw69469.png

Tested it. It is working fine: https://imgur.com/a/bI13q
Either use rButtonTemplate for use the technique posted above to bring the cooldownframe level in line.
